cypress/integration/search.spec.js

```diff
@@ -170,4 +187,25 @@ describe('Search', () => {
             ]);
         });
     });
+
+    describe('Edge Cases', () => {
+        beforeEach(
+            initSearchDataset(
+                'dataset/exotic-search-dataset.csv',
+                'model/exotic-search-model.json',
+            ),
+        );
+
+        it('should have a diacritic insensible text-based search', () => {
+            searchDrawer.openSearchDrawer();
+            searchDrawer.search('sirene');
+            cy.get('.search-result').should('have.length', 1);
+        });
+
+        it('should allow to search for long sentences or descriptions', () => {
+            searchDrawer.openSearchDrawer();
+            searchDrawer.search('Lorem ispum');
```

Comment:
Strange: `Lorem ispum` is absent in `dataset/exotic-search-dataset.csv` (`Lorem ipsum` is present).

Is it on purpose?

Reply:
Is is present on the dataset :o
https://github.com/Inist-CNRS/lodex/pull/817/files#diff-1d1720c228c7e6bddadc0a379259ac80R12

Reply:
Oh, you're right! It's `Lorem ipsum` and not `Lorem ispum`.
Good catch on this one!
The search is too smart :)